libminifi/test/libtest/unit/TestBase.cpp

```diff
@@ -672,11 +672,20 @@ void TestPlan::validateAnnotations() const {
   }
 }
 
+std::vector<std::byte> TestPlan::getContentAsBytes(const core::FlowFile& flow_file) const {
+  const auto content_claim = flow_file.getResourceClaim();
+  const auto content_stream = content_repo_->read(*content_claim);
+  const auto output_stream = std::make_shared<minifi::io::BufferStream>();
+  std::ignore = minifi::InputStreamPipe{*output_stream}(content_stream);
```

Comment:
despite its namespace `internal::pipe` is used in a number of places, so we could use that here as well (I think we should "graduate" it out of the internal namespace in the future)

Reply:
https://github.com/apache/nifi-minifi-cpp/pull/1872/commits/66435037a5a85eb381936ab3a0c708e44c13a73c